ibis/backends/dask/execution/arrays.py

```diff
@@ -53,12 +38,60 @@ def execute_array_column(op, cols, **kwargs):
     )
 
 
+@execute_node.register(ops.ArrayConcat, dd.Series, dd.Series)
+def execute_array_concat_series(op, left, right, **kwargs):
+    df = dd.concat([left, right], axis=1)
+    return df.apply(
+        lambda row: np.concatenate(row), axis=1, meta=dd.utils.make_meta(left),
```

Comment:
In general, if you're not changing the meta at all you can just pass in the meta of the existing dataframe, i.e. `mata=left._meta`.